benchmarks/GraphBLAS-sharp.Benchmarks/Helpers.fs

```diff
@@ -109,35 +141,33 @@ type TEPSColumn() =
 module Utils =
     let getMatricesFilenames configFilename =
         let getFullPathToConfig filename =
-            Path.Combine [|
-                __SOURCE_DIRECTORY__
-                "Configs"
-                filename
-            |] |> Path.GetFullPath
+            Path.Combine [| __SOURCE_DIRECTORY__
+                            "Configs"
+                            filename |]
+            |> Path.GetFullPath
+
 
         configFilename
         |> getFullPathToConfig
         |> File.ReadLines
         |> Seq.filter (fun line -> not <| line.StartsWith "!")
 
     let getFullPathToMatrix datasetsFolder matrixFilename =
-        Path.Combine [|
-            __SOURCE_DIRECTORY__
-            "Datasets"
-            datasetsFolder
-            matrixFilename
-        |]
+        Path.Combine [| __SOURCE_DIRECTORY__
+                        "Datasets"
+                        datasetsFolder
+                        matrixFilename |]
 
     let avaliableContexts =
```

Comment:
```avaliableContexts``` is duplicated: https://github.com/YaccConstructor/GraphBLAS-sharp/pull/36/files#diff-19e4b2c9521563641517ccd062ca9d583c2162016fccce69f427e524bb24dd5bR49